### ISSUE TITLE
The prefix is not reflected in the # HELP and # TYPE lines

### ISSUE THREAD

**Issue by JDA88** [opsdis/aci-exporter#21]
**Describe the bug**
When you use a `prefix` the # HELP and # TYPE lines in the generated metric file does not reflect it.
Effectively rendering those line inefectives

**To Reproduce**
1. Use `prefix: aci_`
2. Look at the generated files
```
# HELP health_ratio Returns health score
# TYPE health_ratio gauge
aci_health_ratio
```

**Expected behavior**
It should be:
```
# HELP aci_health_ratio Returns health score
# TYPE aci_health_ratio gauge
aci_health_ratio
```

**Comment by thenodon** [comment]
@JDA88 thanks for the issue report. This is fixed in branch issue_22 so it can be tested with issue #22.

**Comment by thenodon** [comment]
Merged in #27
